commands/open-shell-container.ts

```diff
@@ -17,7 +16,7 @@ function getEngineTypeShellCommands(engineType: DockerEngineType): string {
     const configOptions: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration('docker');
     switch (engineType) {
         case DockerEngineType.Linux:
-            return configOptions.get('attachShellCommand.linuxContainer', '/bin/sh');
+            return configOptions.get('attachShellCommand.linuxContainer', '/bin/sh -c "[ -e /bin/bash ] && /bin/bash || /bin/sh"');
```

Comment:
This is totally unnecessary, could you remove the default here?  Thanks.

Reply:
I just realized that changing the setting to blank would still not pick up this default. Thanks for the heads up.

Reply:
Should I remove the `powershell`  default too below?